src/ode_solver/ode_solver_factory.cpp

```diff
@@ -125,6 +98,132 @@ void ODESolverFactory<dim,real,MeshType>::display_error_ode_solver_factory(Param
     std::abort();
 }
 
+template <int dim, typename real, typename MeshType>
+std::shared_ptr<ODESolverBase<dim,real,MeshType>> ODESolverFactory<dim,real,MeshType>::create_RungeKuttaODESolver(std::shared_ptr< DGBase<dim,real,MeshType> > dg_input)
+{
+    dealii::ConditionalOStream pcout(std::cout, dealii::Utilities::MPI::this_mpi_process(MPI_COMM_WORLD)==0);
+
+    std::shared_ptr<RKTableauBase<dim,real,MeshType>> rk_tableau = create_RKTableau(dg_input);
```

Comment:
(Read the long comment I left before this one) I think the RK base class should call this in the constructor since it's templated with `n_rk_stages`, rather than it being here